src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -397,11 +402,17 @@ public void periodic() {
       baseLog.log(baseLimit.get());
       scoringLog.log(sourceBreak.get());
     }
-    wristMotor.set(
+
+    FF =
+        wristFeedforward.calculate(
+            Units.rotationsToRadians(wristEncoder.getPosition().getValueAsDouble()), 0);
```

Comment:
I think the velocity shouldn't be zero. We can get the current velocity from the encoder. 
For simulation this requires setting up a new SimDouble for the CANcoder SimDevice